gallery/src/styles/NxList/NxListClickableExample.tsx

```diff
@@ -10,31 +10,41 @@ import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 
 const NxListClickableExample = () =>
   <ul className="nx-list nx-list--clickable">
-    <li className="nx-list__item" tabIndex={0}>
-      <span className="nx-list__text">Action 1</span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link">
+      <button className="nx-list__btn">
+        <span className="nx-list__text">Action 1</span>
+        <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+      </button>
     </li>
-    <li className="nx-list__item" tabIndex={0}>
-      <span className="nx-list__text nx-truncate-ellipsis">
-        This list item should be truncated at the right end edge. youtube weathered network network systemic
-        systema claymore mine voodoo god garage monofilament realism order-flow corporation car footage vinyl.
-      </span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link">
+      <button className="nx-list__btn">
+        <span className="nx-list__text nx-truncate-ellipsis">
+          This list item should be truncated at the right end edge. youtube weathered network network systemic
+          systema claymore mine voodoo
+        </span>
+        <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+      </button>
     </li>
-    <li className="nx-list__item selected" tabIndex={0}>
-      <span className="nx-list__text">This list item is selected</span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link selected" aria-selected="true">
+      <button className="nx-list__btn">
+        <span className="nx-list__text">Action 3</span>
+        <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+      </button>
     </li>
-    <li className="nx-list__item" tabIndex={0}>
-      <span className="nx-list__text">Action 4</span>
-      <span className="nx-list__subtext">
-        This is a fourth line. It includes text that might relate to the top line or might not.
-      </span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link">
+      <button className="nx-list__btn selected" aria-pressed="true">
+        <span className="nx-list__text">Action 4</span>
+        <span className="nx-list__subtext">
+          This list item demonstrates the selected styles
+        </span>
+        <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+      </button>
     </li>
-    <li className="nx-list__item disabled">
-      <span className="nx-list__text">This list item is disabled</span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link" aria-disabled="true">
+      <button className="nx-list__btn disabled">
```

Comment:
Would the `disabled` attribute work here? I'd imagine that might knock out both the styling and the screen reading with a single value rather than needing both the `aria` attr and the class name.

Reply:
> Would the `disabled` attribute work here? I'd imagine that might knock out both the styling and the screen reading with a single value rather than needing both the `aria` attr and the class name.

It does not:
```
Property 'disabled' does not exist on type 'DetailedHTMLProps<LiHTMLAttributes<HTMLLIElement>, HTMLLIElement>
```

Reply:
> It does not:

The button; not the li

Reply:
> > It does not:
> 
> The button; not the li

Ah. Required a SCSS tweak, but works nicely (confirmed in VO): https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/8390a450712248fa2df174655777e24386d22c2c